lib/cog/command/command_resolver.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Cog.Command.CommandResolver do
+
+  alias Cog.Repo
+  alias Cog.Queries.Command
+  alias Cog.Queries.Alias
```

Comment:
@mpeck I've been doing `alias Cog.Queries` and then calling `Queries.Alias`, but that would also work.